AbbreviationAsWordInName/Example1/list-of-projects.properties

```diff
@@ -1,39 +0,0 @@
-# List of GIT repositories to clone / pull for checking with Checkstyle
```

Comment:
Please do not remove files that you are not generating.
Extractor should override files it produces, all other files should be untouched

Reply:
Done. Understood.